mobile/lib/modules/asset_viewer/views/image_viewer_page.dart

```diff
@@ -63,11 +64,8 @@ class ImageViewerPage extends HookConsumerWidget {
           ref.watch(imageViewerStateProvider.notifier).downloadAsset(asset, context);
         },
       ),
-      body: SwipeDetector(
-        onSwipeDown: (_) {

```

Comment:
Why do we remove the dismiss method on swiping down?

Reply:
You have been right. Was able to archieve this using PhotoView's own callback.